AndroidBound/src/main/java/solutions/alterego/androidbound/android/BoundFragmentDelegate.java

```diff
@@ -256,7 +256,7 @@ public void onDestroyView() {
                 && getViewBinder() != null
                 && boundActivityRef.getWindow() != null
                 && boundActivityRef.getWindow().getDecorView() != null) {
-            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView().getRootView());
+            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView());
```

Comment:
yes, we have a reference, I'm just not sure that it's correct that, when one fragment gets removed, the whole activity gets unbound. That would prevent you from changing fragments (think `ViewPager` and what happens when fragment gets removed from it). 

I would prefer for it to be controlled from the activity proper (or `BoundActivityDelegate`).